recipes/ckan-configure.rb

```diff
@@ -41,3 +41,11 @@
 execute "Update supervisor workers if needed" do
     command "supervisorctl update"
 end
+
+if not system("grep 'alias git=' ~/.bash_profile")
+    execute "Add CKAN Git alias to Bash" do
+        command <<-EOS
+            echo "alias git='sudo -u ckan git'" >> ~/.bash_profile
```

Comment:
what is this doing?

Reply:
Amazon Linux 2 is stricter about Git security; even read-only Git operations can't be performed in a directory owned by something else. So this is aliasing `git` to `sudo -u ckan git` to make it easier to work with repositories owned by `ckan`.